configure

```diff
@@ -160,6 +162,7 @@ case "$1" in
     --acle) buildacle=1; shift ;;
     --neon) buildneon=1; shift ;;
     -n | --native) native=1; shift ;;
+    --altivec) noaltivec=0 ; shift ;;
```

Comment:
I feel like this should have a different name, to fit with buildneon/buildacle. Or alternatively with without_optimizations.
Also, why does it deafult to disabled? Functable seems to have runtime detection of altivec, so it seems like we can default it to enabled like we do with most other stuff.

Reply:
It did default to disabled as most older PowerPC machines didn't have Altivec... I can rename the variable if you think "buildaltivec" is better.